cmd/serverless/dependencies_linux_amd64.txt

```diff
@@ -438,6 +438,7 @@ github.com/go-logr/logr
 github.com/go-logr/logr/funcr
 github.com/go-logr/stdr
 github.com/go-viper/mapstructure/v2
+github.com/go-viper/mapstructure/v2/internal/errors
```

Comment:
Where are these error types being used? I'm pretty sure thing change is okay, but I ask because we're working on refactoring the config to not use mapstructure in viper anymore, so it would be good to know what's changing here.

Reply:
good question, I'll look into this. I believe that `go mod tidy` pulled in v2.1 of go-viper/mapstructure over v2.0.0 alpha, which also caused this indirect dependency to come down.

Reply:
Ok, if it's just internal and not used by the code then that's fine. Just wanted to know if any code was using it.

Reply:
- [go.opentelemetry.io/collector/confmap](go.opentelemetry.io/collector/confmap) imports [github.com/go-viper/mapstructure/v2@v2.1.0](github.com/go-viper/mapstructure/v2@v2.1.0)
- [collector/confmap](go.opentelemetry.io/collector/confmap) also imports [github.com/knadh/koanf](github.com/knadh/koanf) which also imports [mapstructure/v2@v2.1.0](github.com/go-viper/mapstructure/v2@v2.1.0)

None of our packages or direct dependencies import [github.com/go-viper/mapstructure/v2/internal/errors](github.com/go-viper/mapstructure/v2/internal/errors); rather [mapstructure/v2/internal/errors](github.com/go-viper/mapstructure/v2/internal/errors) is a sub-module of [go-viper/mapstructure](github.com/go-viper/mapstructure/v2@v2.1.0) so I don't know if there's a way to *not* import it